docs/root/intro/arch_overview/other_features/global_rate_limiting.rst

```diff
@@ -29,3 +29,7 @@ written in Go which uses a Redis backend. Envoy’s rate limit integration has t
   :ref:`Configuration reference <config_http_filters_rate_limit>`
 
 Rate limit service :ref:`configuration <config_rate_limit_service>`.
+
+Note that Envoy also supports :ref:`local rate limiting <config_network_filters_local_rate_limit>`.
```

Comment:
Might be worth it for people's education to have a brief section explaining more deeply why one might need this type of setup. We could have an issue for now and come back to it.

Reply:
What do you want to see beyond what the next sentence says? I can flesh it out.

Reply:
Mainly I was thinking about expanding on the use case. i.e what type of setup might need the filters in working together in order to prevent failure. e.g explain how this might help a setup like Lyft's with mongo.

Reply:
I added a bit more text which will be in the next push.